evap/staff/tools.py

```diff
@@ -368,3 +369,11 @@ def remove_user_from_represented_and_ccing_users(user, ignored_users=None, test_
             cc_user.cc_users.remove(user)
             remove_messages.append(_("Removed {} from the CC users of {}.").format(user.full_name, cc_user.full_name))
     return remove_messages
+
+
+def user_edit_link(user_id):
+    return format_html(
+        '<a href="{}" target=_blank>edit user <span class="fas fa-external-link-alt"></span></a>'.format(
```

Comment:
```suggestion
        '<a href="{}" target=_blank><span class="fas fa-user-pen"> edit user</span></a>'.format(
```

(will work only after #1733 is merged)